src/basic/special.h

```diff
@@ -84,6 +84,8 @@
 #define SPECIAL_VOLATILE_ROOT_SERVICE "systemd-volatile-root.service"
 
 /* Services systemd relies on */
+#define SPECIAL_COREDUMPD_SERVICE "systemd-coredumpd@.service"
+#define SPECIAL_COREDUMPD_SOCKET "systemd-coredumpd.socket"
```

Comment:
hmm, we usually define SPECIAL_xyz defines only for the stuff that is actually special, i.e. has magic meaning for PID 1. But PID 1 doesn't really care about this unit, hence it shouldn't have its own define like this